inference_rules.adoc

```diff
@@ -428,6 +428,79 @@ division allows calibration for quantization and does not allow any retraining.
 The unqualified name “MLPerf” must be used when referring to a Closed Division
 suite result, e.g. “a MLPerf result of 4.5.”
 
+=== Network Division
+
+The Network division inherits all requirements from the Closed division and imposes further constraints.
+In the Network division the SUT is connected to the Loadgen system over a network fabric. The Query Dispatch Library (QDL) component is a submitter-implemented SUT proxy that runs on the Loadgen system. The Network division supports only the Datacenter suite.
```

Comment:
@DilipSequeira Is "suite" the best word we have for the Datacenter and Edge cases?

Reply:
I think we consistently refer to benchmark subsets as suites. It might not be the best word, but at least we are consistent. If you have a better word, I wouldn't object to a rules-wide update.

Reply:
No changes